CMakeLists.txt

```diff
@@ -34,6 +34,12 @@ provide(PACKAGE ubirch VERSION 1.0 BOARD ${BOARD} TARGETS board firmware)
 # add tests
 add_subdirectory(tests)
 
+# copy the config.h.template to config.h in case it is not there; it is ignored by .git!
```

Comment:
Why did you add this here? It should be in tests/modem where the code that uses it is.